documentation.Rmd

```diff
@@ -126,6 +126,9 @@ More recommendations should be organised in a bulleted list:
 #' * [name-repair] documents the details of name repair.
 ```
 
+
+You can link to functions in other packages by using the fully qualified function name, i.e. `pkg::function()`.
```

Comment:
```suggestion
You can link to functions in other packages by using the fully qualified function name, i.e. `[pkg::function()]`.
```

I think?

Reply:
"but note that the package must be Suggested or Imported for the cross-link to resolve"

might be useful

Reply:
I do think it'd be nice if {roxygen2} would automate more there, though https://github.com/r-lib/roxygen2/issues/1470 suggests that might not be in scope.

Reply:
@DavisVaughan I think that's no longer the case as of https://github.com/r-lib/roxygen2/pull/1633. (Which reminds me that maybe I need to do a roxygen2 release)